test/standalone/test-trace-options.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+// Prereqs:
+// Start docker daemon
+//   ex) docker -d
+// Run a mongo image binding the mongo port
+//   ex) docker run -p 27017:27017 -d mongo
+require('../..').start({ samplingRate: 0 });
+
+var common = require('../hooks/common.js');
+
+var assert = require('assert');
+var express = require('../hooks/fixtures/express4');
+var http = require('http');
+
+describe('express + mongo with trace options header', function() {
+  it('should trace when enabled', function(done) {
+    var app = express();
+    app.get('/', function (req, res) {
+      setTimeout(function() {
+        res.send('Hello World');
+      }, 50);
+    });
+    var server = app.listen(common.serverPort, function() {
+      var shouldTraceOptions = [1,3,5,7];
+      var shouldNotTraceOptions = [0,2,4,6];
+      sendRequests(shouldTraceOptions, shouldTraceOptions.length, function() {
```

Comment:
Nit: It would be simpler to concatenate the two arrays and make a single call to `sendRequests`. Or, better, keep it inline.